tests/Elastic.Transport.IntegrationTests/Responses/SpecialisedResponseTests.cs

```diff
@@ -0,0 +1,907 @@
+// Licensed to Elasticsearch B.V under one or more agreements.
+// Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information
+
+using System;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Elastic.Transport.IntegrationTests.Plumbing;
+using Elastic.Transport.Tests.Shared;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Xunit;
+
+namespace Elastic.Transport.IntegrationTests.Responses;
+
+public class SpecialisedResponseTests(TransportTestServer instance) : AssemblyServerTestsBase(instance)
+{
+	private const string Path = "/specialresponse";
+	private const string EmptyJson = "{}";
+	private static readonly byte[] EmptyJsonBytes = [(byte)'{', (byte)'}'];
+	private const string LargeJson = "[{\"_id\":\"672b13c7666cae7721b7f5c8\",\"index\":0,\"guid\":\"f8a9356c-660b-4f4f-a1c2-84048e0599b9\",\"isActive\":false,\"balance\":\"$3,856.23\"," +
```

Comment:
Slight nit use:

```csharp
// language=json
"""
[
 { "_id": "",  
""";
```

That way we can include the json unescaped, pretty formatted and it will highlight properly in IntelliJ editors (not sure about VS :))

Reply:
AFAIK VS supports the highlighting as well!

Reply:
Yeah, good shout. I've pushed that up.